docs/overview/02-getting-started/05-relayer.md

```diff
@@ -91,23 +91,110 @@ main();
 ```
 :::
 
-Once you have all the requirements imported, we will start the verification process by calling a ``POST`` endpoint named ``submit-proof``. We will also need to create a params object with all the necessary information about the proof, which will be sent in the API call. If you want to aggregate the verified proof(want to verify the proof aggregation on connected chains like Sepolia, Base Sepolia etc) check the code snippets with aggregation.
+Once you have all the requirements imported, we will start by registering our verification key. We can do this by calling a ``GET`` endpoint named ``register-vk`` on our relayer service. We will also need to create a params object with all the necessary information about the verification key, which will be sent in the API call. Once we register the verification key, we will store the ``vkHash`` in a json file which can be used in subsequent API verification calls.
+
+<Tabs groupId="register-vk">
+<TabItem value="circom" label="Circom">
+```js
+if(!fs.existsSync("circom-vkey.json")){
+    try{
+        const regParams = {
+            "proofType": "groth16",
+            "proofOptions": {
+                "library": "snarkjs",
+                "curve": "bn128"
+            },
+            "vk": key
+        }
+        const regResponse = await axios.post(`${API_URL}/register-vk/${process.env.API_KEY}`, regParams);
+        fs.writeFileSync(
+            "circom-vkey.json",
+            JSON.stringify(regResponse.data)
+        );
+    }catch(error){
+        fs.writeFileSync(
+            "circom-vkey.json",
+            JSON.stringify(error.response.data)
+        );
```

Comment:
Is the error written to the file to reference later? 
Might be better to write to a separate file like `circom-vkey.error.json` to prevent it from overwriting

Reply:
And all the other places this is done too